src/dom.js

```diff
@@ -383,3 +383,19 @@ export function getDataParamsFromAttributes(element, opt_computeParamNameFunc) {
   }
   return params;
 }
+
+
+/**
+ * Whether the element has been already fully parsed by the browser.
```

Comment:
I don't think this is a good name since we can't really judge "parsed" in all different cases. But we can name it something like "hasNextNodeInDocumentOrder" or such that directly explains what it does.